### PR TITLE
Run Firestore tests against Emulators where possible.

### PR DESCRIPTION
**Pull request by wu-hui**
Firestore tests run against Firestore emulator except:
* When run from Test Lab
* Windows (setup Firestore emulator does not work out-of-box), will address in future PR.

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -739,11 +739,26 @@ jobs:
...
       - name: Setup integration test deps
         run: |
           pip install -r scripts/gha/requirements.txt
           python scripts/gha/restore_secrets.py --passphrase "${{ secrets.TEST_SECRET }}" --artifact testapps
-      - name: Run Desktop integration tests
+      - name: Run Desktop integration tests (Non-windows)
+        if: ${{ !startsWith(matrix.os, 'windows') }}
+        run: python scripts/gha/desktop_tester.py --testapp_dir testapps --logfile_name "desktop-${{ matrix.os }}-${{ matrix.ssl_variant }}"
+        env:
+          USE_FIRESTORE_EMULATOR: true
```
Comment (sunmou99) [inline .github/workflows/integration_tests.yml:759]: I don't know all the detail. But I was wondering why Non-windows need this env. 
Isn't `intent.putExtra("USE_FIRESTORE_EMULATOR", "true")` enough to set the env?

Comment (wu-hui) [reply .github/workflows/integration_tests.yml:759]: It is enough for Android. This is running desktop tests.

Comment (sunmou99) [reply .github/workflows/integration_tests.yml:759]: My bad. Then why Windows doesn't have this env?

**Review by sunmou99 — APPROVED** [review]
LGTM. 
Please follow up on Firestore Emulator on Windows.

**Comment by wu-hui** [comment]
> LGTM.
> Please follow up on Firestore Emulator on Windows.

Please take another look, windows is now supported.

**Review by sunmou99 — APPROVED** [review]
LGTM

**Pull request merged**